ssl/ssl_ciph.c

```diff
@@ -1643,6 +1643,7 @@ STACK_OF(SSL_CIPHER) *ssl_create_cipher_list(SSL_CTX *ctx,
         }
```

Comment:
It is interesting that 1.1.1 does not seem to have the "Don't include any TLSv1.3 ciphers that are disabled"
code above

Reply:
In 3.0 it is possible to have TLSv1.3 ciphersuites that are known and recognised by libssl, but can't actually be used at runtime because an implementation of the underlying cipher/hash is not available (i.e. because of a lack of provider support).

In 1.1.1 this is not the case. Either the ciphersuites are known and compiled in and therefore available, or we don't even know about the ciphersuites.